codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -2540,12 +2552,41 @@ simulationControls w
     | playbackSpeed w == 0 = [PlayButton, StepButton, SpeedSlider]
     | otherwise = [PauseButton, SpeedSlider]
 
+debugSimulationControls :: Wrapped [w] -> [Control [w]]
```

Comment:
Since these are used for both simulations and interactions, is there a better name?  Perhaps statefulDebugControls?  That's not great, but you get the idea...

Reply:
I used ```statefulDebugControls```, but if there's a better name, can always change it.